cmd/installer/pkg/bootloader/syslinux/syslinux.go

```diff
@@ -253,3 +214,82 @@ func WriteSyslinuxCfg(base, path string, syslinuxcfg *Cfg) (err error) {
 
 	return nil
 }
+
+func writeUEFIFiles() (err error) {
+	efiDir := filepath.Join(constants.BootMountPoint, "EFI", "BOOT")
+	if err = os.MkdirAll(efiDir, 0700); err != nil {
+		return err
+	}
+
+	input, err := ioutil.ReadFile(syslinuxefi)
+	if err != nil {
+		return err
+	}
+
+	err = ioutil.WriteFile(efiDir+"/BOOTX64.EFI", input, 0600)
+	if err != nil {
+		return err
+	}
+
+	input, err = ioutil.ReadFile(ldlinuxe64)
```

Comment:
nit:  maybe use `io.Copy()` to avoid reading whole files into RAM?

Reply:
Yeah I opted for simplicity here since these files are kbs, but it still bugs me that we read the whole thing into memory. I'll clean up.

Reply:
Fixed.